packages/testing/src/api/index.ts

```diff
@@ -64,6 +72,32 @@ export function initializeTestApp(options: any): admin.app.App {
   );
 }
 
+export type FirestoreTestAppOptions = {
+  projectId: string;
+  auth: Object;
+};
+export function initializeFirestoreTestApp(
+  options: FirestoreTestAppOptions
+): FirebaseApp {
+  const header = {
+    alg: 'RS256',
+    kid: 'fakekid'
+  };
+  const fakeToken = [
+    base64.encodeString(JSON.stringify(header), /*webSafe=*/ false),
+    base64.encodeString(JSON.stringify(options.auth), /*webSafe=*/ false),
+    'fakesignature'
+  ].join('.');
+  const app = firebase.initializeApp(
+    { projectId: options.projectId },
+    'app-' + new Date().getTime() + '-' + Math.random()
```

Comment:
You can make this a little more intuitive if you add a local variable that will help let users know that this is the `appName`.